app/[locale]/institute/sections/sports/page.tsx

```diff
@@ -13,5 +25,385 @@ export default async function Sports({
     where: (section, { eq }) => eq(section.urlName, 'sports'),
   }))!;
 
-  return <WorkInProgressStatus locale={locale} />;
+  const roadmapTableData = [
+    [{ span: 2 }, { span: 8, className: 'bg-success' }, { span: 2 }],
+    [
+      { span: 2, className: 'bg-primary-500' },
+      { span: 8 },
+      { span: 2, className: 'bg-primary-500' },
+    ],
+    [{ span: 4 }, { span: 1, className: 'bg-secondary-100' }, { span: 7 }],
+    [{ span: 8 }, { span: 1, className: 'bg-[#800080]' }, { span: 3 }],
+    [{ span: 9 }, { span: 1, className: 'bg-secondary-500' }, { span: 2 }],
+  ];
+
+  return (
+    <>
+      <ImageHeader
+        title={text.title}
+        headings={[
+          { label: text.headings.about, href: '#about' },
+          { label: text.headings.swimmingPool, href: '#swimming-pool' },
+        ]}
+        src="institute/sports/header.jpg"
+      />
+      <section className="container" id="about">
+        <Heading
+          glyphDirection="ltr"
+          heading="h2"
+          id="#about"
+          text={text.about.title}
+        />
+        <Table scrollAreaClassName="mb-10">
+          <TableHeader>
+            <TableRow>
+              <TableHead>{text.about.profTableTitle.name}</TableHead>
+              <TableHead>{text.about.profTableTitle.details}</TableHead>
+            </TableRow>
+          </TableHeader>
+          <TableBody>
+            <Suspense
+              fallback={
+                <TableRow>
+                  <TableCell colSpan={2} rowSpan={2}>
+                    Loading...
+                  </TableCell>
+                </TableRow>
+              }
+            >
+              <DelayedStaff
+                id={section.id}
+                headId={section.headFacultyId}
+                localeText={{
+                  email: text.about.email,
+                  phone: text.about.phone,
+                  headPosition: text.about.headPosition,
+                }}
+              />
+            </Suspense>
+          </TableBody>
+        </Table>
+        <p className="whitespace-pre-line">{section?.aboutUs}</p>
+      </section>
+      <section className="container space-y-4" id="swimming-pool">
+        <Heading
+          glyphDirection="rtl"
+          heading="h2"
+          id="#swimming-pool"
+          text={text.swimmingPool.title}
+        />
+        <h5>{text.swimmingPool.welcome}</h5>
```

Comment:
Absolutely not

Reply:
?

Reply:
Don't put that welcome message there. The user is visiting the website, not the swimming pool

Reply:
Just copied the existing page